### PR TITLE
There should be a way to navigate between tabs of a PR #869

### LINKED ISSUE #869: There should be a way to navigate between tabs of a PR
as per title

### PR DESCRIPTION
**Pull request by rd1992**
Fixes #869

### CODE REVIEW

File: src/main/java/browserview/BrowserComponent.java [hunk 1]
```diff
@@ -487,9 +487,9 @@ public String getCurrentUrl() {
         return driver.getCurrentUrl();
     }
 
-    public void switchToConversationTab() {
+    public void switchToTab(String tabName) {
```
Comment (dariusf) [inline src/main/java/browserview/BrowserComponent.java]: Could this string be an enum instead? We don't want callers of this method to have to know how GitHub's DOM elements are named. The enum can be in GitHubURL and have a toString method which generates the string.

File: src/main/java/util/GitHubURL.java [hunk 2]
```diff
@@ -49,7 +49,7 @@ public static boolean isUrlIssue(String url) {
     }
     
     public static boolean isPullRequestLoaded(String url)  {
-        return (url.endsWith("/commits") || url.endsWith("/files"));
+        return url.matches("https://github.com/([^/]+)/([^/]+)/pull/([0-9]+)([/commits,/files]*)");
```
Comment (dariusf) [inline src/main/java/util/GitHubURL.java]: The last part doesn't look right. `[/commits,/files]*` is a character class, so it will match things like `cc`, `,,` -- 0 or more of any character in the brackets.

Also we don't have to parenthesise groups that we're not going to refer to. That usually makes the regex a tiny bit faster.

This should probably have tests, given how mistakes in regexes can sometimes be very subtle.

Comment (damithc) [inline src/main/java/util/GitHubURL.java]: When using regular expressions, use a comment to explain the intended meaning of the expression (e.g. 8 character strings ending with a number) and write test cases with a few examples of matching strings and boundary cases.

**Comment by rd1992** [comment]
@dariusf final review before i merge it? (took ages to pass the travis test)

**Comment by dariusf** [comment]
Looks good. Nice work!
